First/dailyExercises/InsertString.java

```diff
@@ -0,0 +1,29 @@
+package firstPackage.First.dailyExercises;
+
+public class InsertString {
+    // Function to insert string
+    public static String insertString(String originalString, String stringToBeInserted, int var) {
```

Comment:
Please give proper variable name for var.

Reply:
DONE !